Sources/StreamChat/Controllers/MemberListController/MemberListController.swift

```diff
@@ -95,9 +95,8 @@ public class ChatChannelMemberListController: DataController, DelegateCallable,
         )
     }
 
-    private func createMemberListObserver() -> ListDatabaseObserverWrapper<ChatChannelMember, MemberDTO> {
+    private func createMemberListObserver() -> BackgroundListDatabaseObserver<ChatChannelMember, MemberDTO> {
```

Comment:
⚠️  This actually needs to the the Main thread Database observer, otherwise, user mentions won't work properly

Reply:
I'll change it back but we should tackle this in following PRs, I bet it is related to the issue of accessing initial items.